srcpkgs/AusweisApp2/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'AusweisApp2'
+pkgname=AusweisApp2
+version=1.22.1
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config qt5-qmake qt5-host-tools"
+makedepends="qt5-devel qt5-tools-devel pcsclite-devel qt5-svg-devel qt5-websockets-devel qt5-declarative-devel openssl-devel qt5-quickcontrols2-devel"
```

Comment:
Please break this at ~80 columns

Reply:
and use a leading whitespace in the new line

Reply:
I did break at 80 columns and got a leading tabular in the new line with commit 
6a31d0e

Reply:
indeed, and no template does that. for style consistency, i asked you to use a space instead of tab.

Reply:
now I understand you :-)
Was a little irritated will fix it in 10 minutes

Reply:
why I got the xlint error indent with tabs now?